pages/common/scrapy.md

```diff
@@ -0,0 +1,31 @@
+# scrapy
+
+> Open Source framework for crawling websites written in python.
+
+- Create a project:
+
+`scrapy startproject {{project_name}}`
+
+- Create a spider:
+
+`scrapy genspider {{spider_name}} {{website_domain}}`
+
+- Edit spider:
+
+`scrapy edit {{spider_name}}`
+
+- Run spider:
+
+`scrapy crawl {{spider_name}}`
+
+- Fetch a webpage as scrapy sees it and print source in stdout:
+
+`scrapy fetch {{url}}`
+
+- View a webpage as scrapy sees it, opens it in browser(disable javascript before):
```

Comment:
Ok, so adding that would probably be a good idea, e.g. "disable javascript in the browser to match what scrapy sees". How about something like "View a webpage in the browser as scrapy sees it (disable javascript before to match what scrapy sees)"? I'm not entirely sold on this particular wording, but that's the general structure I feel would be clearest.